src/main/java/org/littleshoot/proxy/impl/ProxyToServerConnection.java

```diff
@@ -269,7 +273,9 @@ protected boolean isContentAlwaysEmpty(HttpMessage httpMessage) {
             return HttpMethod.HEAD.equals(currentHttpRequest.getMethod()) ?
                     true : super.isContentAlwaysEmpty(httpMessage);
         }
-    };
+    }
+
+    ;
```

Comment:
There are a lot of whitespace changes like these that are not desirable and add noise. Can you clean them up?